src/app/breadcrumbs/concept-breadcrumb/concept-breadcrumb.jsx

```diff
@@ -8,12 +8,19 @@ import {
   searchConcepts
 } from '../../../api/search-fulltext-api/concepts';
 import { getTranslateText } from '../../../lib/translateText';
+import translations from '../../../lib/localization';
 
 const memoizedSearchConcepts = _.memoize(searchConcepts);
 
 const PureConceptBreadcrumb = props => {
   const { conceptItem } = props;
-  return <span>{getTranslateText(_.get(conceptItem, 'prefLabel'))}</span>;
+  return (
+    <span>
+      {getTranslateText(
+        _.get(conceptItem, 'prefLabel') ?? translations.breadcrumb.notFound
```

Comment:
Pass på her, siden dette er ikke en tsx fil. Anbefaler deg å konvertere til tsx, da kan du gjøre ting enda enklere:

```suggestion
        conceptItem?.prefLabel ?? translations.breadcrumb.notFound
```